src/components/ContinuationIn/UnlimitedLiabilityCorporationInformation.vue

```diff
@@ -0,0 +1,258 @@
+    this.$refs.fileUploadPreview.clickFileInput()
+  }
+  /**
+   * Called when FileUploadPreview tells us whether a file is valid.
+   * This is called right before the File Selected event.
+   */
+  onFileValidity (valid: boolean): void {
+    this.fileValidity = valid
+  }
+
+  /**
+   * Called when FileUploadPreview tells us about a new or cleared file.
+   * This is called right after the File Validity event.
+   * @param file the file to add
+   */
+  async onFileSelected (file: File): Promise<void> {
+    /** Returns True if given string contains only Latin 1 (ie, ISO8859-1) characters. */
+    function isValidLatin1 (str: string): boolean {
+      // eslint-disable-next-line no-control-regex
+      return !/[^\u0000-\u00ff]/g.test(str)
+    }
+
+    // verify that file is specified and is valid
+    if (file && this.fileValidity) {
+      // verify file name encoding
+      if (!isValidLatin1(file.name)) {
+        // set error message
+        this.customErrorMessage = 'Invalid character in file name.'
+        return // don't add to array
+      }
+
+      // try to upload to Minio
+      let psu: PresignedUrlIF
+      try {
+        this.isDocumentLoading = true
+        psu = await this.getPresignedUrl(file.name)
+        const res = await this.uploadToUrl(psu.preSignedUrl, file, psu.key, this.getKeycloakGuid)
+        if (!res || res.status !== StatusCodes.OK) throw new Error()
+      } catch {
+        // set error message
+        this.customErrorMessage = this.UPLOAD_FAILED_MESSAGE
+        return // don't add to array
+      } finally {
+        this.isDocumentLoading = false
+      }
+
+      // add properties reactively to business object
+      this.$set(this.getExistingBusinessInfo, 'affidavitFile', {
+        name: file.name,
+        lastModified: file.lastModified,
+        size: file.size
+      } as File)
+      this.$set(this.getExistingBusinessInfo, 'affidavitFileKey', psu.key)
+      this.$set(this.getExistingBusinessInfo, 'affidavitFileName', file.name)
+
+      // user has changed something
+      this.setHaveChanges(true)
+    }
+  }
+
+  /**
+   * Called when user clicks Remove button.
+   * May also be called by parent component to remove the file info.
+   */
+  onRemoveClicked (): void {
+    // delete file from Minio, not waiting for response and ignoring errors
+    this.deleteDocument(this.getExistingBusinessInfo.affidavitFileKey).catch(() => null)
+
+    // delete properties reactively
+    this.$delete(this.getExistingBusinessInfo, 'affidavitFile')
+    this.$delete(this.getExistingBusinessInfo, 'affidavitFileKey')
+    this.$delete(this.getExistingBusinessInfo, 'affidavitFileName')
+
+    // clear any existing error message
+    this.customErrorMessage = ''
+
+    // user has changed something
+    this.setHaveChanges(true)
+  }
+
+  /** Watches for changes to affidavit file object and emits form validity. */
+  @Watch('getExistingBusinessInfo.affidavitFile', { deep: true, immediate: true })
```

Comment:
Use "deep" in case any of the objet properties change.

Use "immediate" so this fires when the component is created/mounted. This way, this component its validity initially.